typings/custom-scope-defs.d.ts

```diff
@@ -81,4 +81,7 @@ interface ICustomScope extends ng.IScope {
     staffTop?: number;
     submitAnswer?: (() => void);
     topPositionForCenterOfTopStaffLine?: number;
+
+    // apply-validation.directive.ts
+    $ctrl: any;
```

Comment:
This seems like a very general name that may affect more things than we want it to. Should we make the name more specific?

Reply:
We are using `$ctrl` throughout the codebase for accessing values bound to controller (following the convention from the upgrade guide). In the apply-validation directive, `scope` is used in `link` function and we use `scope.$ctrl.validators()` to access the validators. So, we need to add `$ctrl` in custom defs. So, do you want to update the naming convention to use specific `controllerAs` names instead of `$ctrl`?

I am sorry but I am not sure if I understood your comment correctly. Please do let me know if there are other queries.

Reply:
Oh, I see. I guess my question is -- let's say some other file uses $ctrl. Is "any" an appropriate type there too, or should it be more specific?

In general I get a bit worried whenever I see "any". Can it be made more specific?

Reply:
I don't think we can make type more specific for `$ctrl` since it used as a controller name for binding properties to controller and different controllers have different properties bound to them. So, we will need to rename all controllers and assign specific type to each of them for removing the any type here.

Reply:
I see, OK. Yeah, on balance, let's keep it as you have it. LGTM then.

Thanks!